README.md

```diff
@@ -25,3 +25,13 @@ Want to see other mobile projects Mozilla works on? A few others are:
 - [Lockbox](https://github.com/mozilla-lockbox/)
 - [Firefox Reality (for VR)](https://github.com/mozillareality/firefoxreality)
 - [Firefox Rocket](https://github.com/mozilla-tw/Rocket)
+
+## Improving this documentation
+Didn't understand our documentation? Please file an issue or open a pull request:
+we encourage all contributions!
```

Comment:
Is filing an issue easier than pinging a specific person on irc? I guess probably. I'm afraid that we never triage the issues here though, unless you want to primarily do that or maybe explicitly include people in the repo to get updates?

Reply:
I'm watching the repository so I should see them.